alacritty_terminal/src/event.rs

```diff
@@ -38,6 +38,9 @@ pub enum Event {
     /// Write some text to the PTY.
     PtyWrite(String),
 
+    /// Request to write the text area size.
+    TextAreaSize(Arc<dyn Fn(WinSize) -> String + Sync + Send + 'static>),
```

Comment:
Our color variant is called `ColorRequest`, should we maybe rename this to `TextAreaSizeRequest` for consistency?